catalog/task/update-ocp-tag/0.1/README.md

```diff
@@ -0,0 +1,14 @@
+# update-ocp-tag
+
+Tekton task to update version tag of FBC pull-spec
+ `fromIndex`, `targetIndex` and `binaryImage` with the 
+ provided OCP version from the `get-ocp-version` task.
```

Comment:
if it is updating it based on a parameter value, the task doesn't really care what other task it came from